server/tailsql/tailsql.go

```diff
@@ -461,19 +463,17 @@ func (s *Server) queryContext(ctx context.Context, caller string, q Query) (*dbR
 				q.Query = real
 			}
 
-			rows, err := tx.QueryContext(fctx, q.Query)
+			rows, err := db.Query(fctx, q.Query)
 			if err != nil {
 				return nil, err
 			}
 			defer rows.Close()
 
-			cols, err := rows.ColumnTypes()
+			cols, err := rows.Columns()
```

Comment:
`out.Columns, err =`?

Reply:
I'd rather not, because then I have to replace all the nice compact `cols` references below, and that makes it harder to read.